cellProps missing in TypeScript definitions

After upgrading to `2.2.1`, I'm getting the following error from TypeScript:

```
src/Devices/Devices.tsx:122:9 - error TS2322: Type '({ header: Element; cell: (d: SelectableDevice) => Element; width: number; cellProps: { padding: string; }; } | { header: string; name: string; } | { header: string; name: string; width: number; } | { ...; } | { ...; } | { ...; } | { ...; })[]' is not assignable to type 'IMuiVirtualizedTableColumn<SelectableDevice>[]'.
  Type '{ header: Element; cell: (d: SelectableDevice) => Element; width: number; cellProps: { padding: string; }; } | { header: string; name: string; } | { header: string; name: string; width: number; } | { ...; } | { ...; } | { ...; } | { ...; }' is not assignable to type 'IMuiVirtualizedTableColumn<SelectableDevice>'.
    Type '{ header: Element; cell: (d: SelectableDevice) => Element; width: number; cellProps: { padding: string; }; }' is not assignable to type 'IMuiVirtualizedTableColumn<SelectableDevice>'.
      Object literal may only specify known properties, and 'cellProps' does not exist in type 'IMuiVirtualizedTableColumn<SelectableDevice>'.

122         cellProps: { padding: 'checkbox' },
            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  node_modules/mui-virtualized-table/index.d.ts:81:3
    81   columns: Array<IMuiVirtualizedTableColumn<TRow>>;
         ~~~~~~~
    The expected type comes from property 'columns' which is declared here on type 'IntrinsicAttributes & IntrinsicClassAttributes<MuiVirtualizedTable<SelectableDevice>> & Readonly<PropsWithChildren<IMuiVirtualizedTableProps<SelectableDevice>>>'


Found 1 error.
```

It seems like the `cellProps` property is missing in the TypeScript definitions for `IMuiVirtualizedTableColumn`.

--- a/index.d.ts
+++ b/index.d.ts
@@ -38,6 +38,8 @@ export interface IMuiVirtualizedTableColumn<TRow = any> {
    * Can be a `number`, straight pixel width, or percentage string like `40%`
    */
   width?: number | string;
+
+  cellProps?: ICellPropsProducer<TRow>;
 }
 
 export interface IMuiVirtualizedTableProps<TRow> {
@@ -67,13 +69,11 @@ export interface IMuiVirtualizedTableProps<TRow> {
    */
   columnWidth?:
     | number
-    | ((
-        column: {
-          index: number;
-          columns: Array<IMuiVirtualizedTableColumn<TRow>>;
-          width: number;
-        }
-      ) => number);
+    | ((column: {
+        index: number;
+        columns: Array<IMuiVirtualizedTableColumn<TRow>>;
+        width: number;
+      }) => number);
 
   /**
    * Defines the columns in the table

